public/javascripts/indexController.js

```diff
@@ -19,10 +19,6 @@ var indexController = function($scope, $http, $location, $state) {
 
   $http.get('/api/allStudents').then(function(data) {
     $scope.students = data.data.allStudents;
-    $scope.juniorSpring = data.data.juniorSpring;
-    $scope.babySpring = data.data.babySpring;
-    $scope.littleSpring = data.data.littleSpring;
```

Comment:
Yeah, I don't think we were using these. But it might be useful for future to view all the cohort students